website/blog/2024-02-29-Deprecation-notice-for-cordova-project-type.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Deprecation notice for Cordova project type"
+author: Prasanth Reddy
+---
+---
+
+## Deprecation notice for Cordova project type
+
+WaveMaker’s support for developing mobile applications was launched in 2015 with Cordova as the underlying framework. But with multiple advancements, in 2022, we released support for creating React Native mobile applications. 
+
+For us to serve our customers better, we are moving our mobile focus to React Native only. With the release of 11.6.0, we will start providing minimal support to Cordova projects. 
```

Comment:
considering the benefits/advantages react native provides, we are moving our mobile focus to React native only to serve our customers better.

Reply:
we are going to discontinue support for creating new cordova projects.